.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode7.3
+osx_image: xcode8.0
```

Comment:
I've also found that we cals also use instead of 
`-sdk iphonesimulator9.3`
this
`-destination 'platform=iOS Simulator,name=iPhone 6s' CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO ONLY_ACTIVE_ARCH=NO`